src/EditorFeatures/CSharpTest2/Recommendations/WhenKeywordRecommenderTests.cs

```diff
@@ -191,15 +191,15 @@ public async Task TestForCatchClause_NotAfterFilter3()
         public async Task TestForSwitchCase_NotInEmptySwitchStatement() =>
             await VerifyAbsenceAsync(AddInsideMethod(@"switch (1) { $$ }"));
 
-        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/40015"), Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
```

Comment:
Should these also have prototype comments?

Reply:
No, since we have an open issue to track this.

---
In reply to: [352880470](https://github.com/dotnet/roslyn/pull/39913#discussion_r352880470) [](ancestors = 352880470)